metta/metta_io.py

```diff
@@ -122,7 +166,7 @@ def _store_matrix(metadata, df_data, title, directory, format='hd5'):
             yaml.dump(metadata, stream)
 
         if format == 'hd5':
-            hdf = pd.HDFStore(directory + '/' + title + '.h5')
+            hdf = pd.HDFStore(directory + '/' + title + '.h5', complevel=5, complib="zlib")
```

Comment:
Can you check what your performance is for `zlib` and `blosc` filters for sizes, compression and decompression speed for a typical matrix. I have a feeling blosc will be much more efficient.